composeApp/src/main/kotlin/io/github/couchtracker/ui/screens/viewing/DateTimeSection.kt

```diff
@@ -0,0 +1,332 @@
+    section(Text.Resource(R.string.date_and_time)) {
+        val transition = updateTransition(sectionState.dateTime)
+        transition.Crossfade(
+            contentKey = { dt ->
+                dt?.category == DateAndTimeSectionChoices.CUSTOM
+            },
+        ) { dt ->
+            if (dt?.category == DateAndTimeSectionChoices.CUSTOM) {
+                CustomDateTimeRow(
+                    selectedDateTime = dt.dateTime,
+                    setCustomDialogVisibility = { sectionState.customDateDialogVisibility = it },
+                    deselect = { sectionState.dateTime = null },
+                )
+            } else {
+                LazyRow(
+                    contentPadding = PaddingValues(horizontal = 16.dp),
+                    horizontalArrangement = Arrangement.spacedBy(8.dp),
+                ) {
+                    items(DateAndTimeSectionChoices.entries) { category ->
+                        FilterChip(
+                            selected = category == dt?.category,
+                            onClick = {
+                                when (category) {
+                                    dt?.category -> {
+                                        sectionState.dateTime = null
+                                    }
+
+                                    DateAndTimeSectionChoices.CUSTOM -> {
+                                        sectionState.customDateDialogVisibility = DATE
+                                    }
+
+                                    else -> {
+                                        sectionState.dateTime = DateAndTimeValue(
+                                            category,
+                                            category.default(Clock.System.now(), TimeZone.currentSystemDefault(), approximateVideoRuntime),
+                                        )
+                                    }
+                                }
+                            },
+                            colors = FilterChipDefaults.filterChipColors(
+                                selectedContainerColor = MaterialTheme.colorScheme.primaryContainer,
+                            ),
+                            label = { Text(category.displayName.string()) },
+                        )
+                    }
+                }
+            }
+        }
+    }
+}
+
+@OptIn(ExperimentalLayoutApi::class)
+@Composable
+private fun CustomDateTimeRow(
+    selectedDateTime: PartialDateTime,
+    setCustomDialogVisibility: (DateTimeSectionState.CustomDateDialogVisibility) -> Unit,
+    deselect: () -> Unit,
+) {
+    val localDate = selectedDateTime.local
+    val dateString = when (localDate) {
+        is PartialDateTime.Local.Year -> localDate.localized(YearSkeleton.NUMERIC).string()
+        is PartialDateTime.Local.YearMonth -> localDate.localized(YearSkeleton.NUMERIC, MonthSkeleton.NUMERIC).string()
```

Comment:
Can we use `MonthSkeleton.WIDE` (or `MonthSkeleton.ABBREVIATED`) here? I think it's clearer.